src/plugins/lua/luaplugin.cpp

```diff
@@ -199,14 +228,20 @@ static bool includeTile(const Tile *tile)
 }
 
 void LuaPlugin::writeTileset(LuaTableWriter &writer, const Tileset *tileset,
-                             unsigned firstGid)
+                             unsigned firstGid, bool standalone)
```

Comment:
I've just realised that I've got the semantics of this bool backwards. Please disregard as this will certainly be rectified if these changes move forward.